app/views/comment_types/edit.html.erb

```diff
@@ -1,3 +1,3 @@
-<main class="comment_type_edit">
+<main class="comment-type-edit">
```

Comment:
I am personally a big fan of this class name, and then making all of the styling for this page _scoped_ within this class. :)